docker-compose/community-docker-compose.yml

```diff
@@ -72,7 +84,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: alfresco/alfresco-search-services:2.0.5
+    image: docker.io/alfresco/alfresco-search-services:2.0.7-A4
```

Comment:
we probably don't want this, community version should always use dockerhub (I remember there was a test for it - it didn't run because you have merge conflicts)

Reply:
I'm not sure either we should have an -A release here. I see no plan to release 2.0.7 at the end of Oxford. @dsibilio can you clarify what is 2.0.7 and when is it planned for?

Reply:
I proposed to Andy Leach that we'd release IE/SS 2.0.7 to address the Rocky Linux 8.7 supported platform upgrade (planned to be completed and released in Oxford). I'm expecting it to be released at the end of Oxford, unless I'm told otherwise later on during the current train.